src/app/task/shared/services/task.service.ts

```diff
@@ -1,36 +1,33 @@
 import { Injectable } from "@angular/core";
 import { HttpClient } from "@angular/common/http";
 import { Task } from "../models/task.model";
+import { Observable } from "rxjs";
 
 @Injectable({
   providedIn: 'root'
 })
 export class TaskService {
-  taskList : Task[] = new Array();
+  BASE_URL = 'http://localhost:3000/api/tasks/';
```

Comment:
You can create a constants.ts file in a shared folder, where all your constants will be defined and import them to other components/services etc when needed

Reply:
I agree

Reply:
Do i have to make it a component or can i store it in .env file?